client/components/theme/style.scss

```diff
@@ -403,3 +403,20 @@ $theme-info-height: 54px;
 .theme__tooltip {
 	max-width: 300px;
 }
+
+.theme__info-soft-launched {
+	position: absolute;
+	top: 36px;
+	left: -12px;
+	z-index: 100000;
```

Comment:
Could we adjust the z-index value? Otherwise, you might see something like the screenshot below

![image](https://user-images.githubusercontent.com/13596067/197958469-0055a25d-1d49-4cf2-9973-9c3e3d97697b.png)

Reply:
That should be fixed now! Thank you!